ktor-server/ktor-server-core/jvm/src/io/ktor/server/http/content/LocalFileContent.kt

```diff
@@ -27,9 +27,9 @@ public class LocalFileContent(
     init {
         val lastModifiedVersion = file.lastModified()
         versions += LastModifiedVersion(
-            if (lastModifiedVersion == 0L)
+            if (lastModifiedVersion == 0L) {
```

Comment:
please exctract this check outside of constructor call

Reply:
Done